docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -1755,19 +1967,89 @@ Maximal number of requests through a single keep-alive connection until it will
 
 **Example**
 
-``` xml
+```xml
 <max_keep_alive_requests>10</max_keep_alive_requests>
 ```
 
+## ldap_servers
+
+List LDAP servers with their connection parameters here to later:
+- use them as authenticators for dedicated local users, who have an 'ldap' authentication mechanism specified instead of 'password'
+- use them as remote user directories.
+
+The following parameters are accepted:
+
+| Parameter                      | Description                                                                                                                                                                                                                                                                                                                                                                                                                              |
+|--------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `host`                         | LDAP server hostname or IP, this parameter is mandatory and cannot be empty.                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
LDAP auth is obscure, assuming you copypasted from config.xml. Didn't check deeper.